loopy/statistics.py

```diff
@@ -724,17 +730,21 @@ class MemAccess(ImmutableRecord):
        work-group executes on a single compute unit with all work-items within
        the work-group sharing local memory. A sub-group is an
        implementation-dependent grouping of work-items within a work-group,
-       analagous to an NVIDIA CUDA warp.
+       analogous to an NVIDIA CUDA warp.
 
     .. attribute:: kernel_name
 
         A :class:`str` representing the kernel name where the operation occurred.
+
+    .. attribute:: tags
+
+        A :class:`frozenset` of tags to the operation.
     """
 
     def __init__(self, mtype=None, dtype=None, lid_strides=None, gid_strides=None,
                  direction=None, variable=None,
                  *, variable_tags=None,
-                 count_granularity=None, kernel_name=None):
+                 count_granularity=None, kernel_name=None, tags=None):
```

Comment:
Would it make sense to default this to an empty `frozenset` instead? (Here and elsewhere?)